src/google/protobuf/any.pb.h

```diff
@@ -167,7 +167,7 @@ class LIBPROTOBUF_EXPORT Any : public ::google::protobuf::Message /* @@protoc_in
   ::google::protobuf::internal::ArenaStringPtr value_;
   mutable int _cached_size_;
   ::google::protobuf::internal::AnyMetadata _any_metadata_;
-  friend struct LIBPROTOBUF_EXPORT protobuf_google_2fprotobuf_2fany_2eproto::TableStruct;
+  friend struct protobuf_google_2fprotobuf_2fany_2eproto::TableStruct;
```

Comment:
the suspicuous change is due to https://github.com/google/protobuf/commit/b4b855cde4197de605d463630af9f0ea89583bca
generate_descriptor_proto.sh was not run after this change